libs/designsystem/src/lib/scss/_global-styles.scss

```diff
@@ -64,6 +75,12 @@ ion-modal.kirby-overlay {
     --border-radius: 0;
     --height: auto;
 
+    &.invert-colors {
+      --background: #{get-color('background-color-contrast')};
+      --kirby-modal-color: #{get-color('background-color')};
+      --ion-text-color: #{get-color('background-color')};
+    }
```

Comment:
As per tech refinement, this should be handled by exposing css custom props for background + text color and let the target project set those